### PR TITLE
Fix network aircraft primary key

### PR DESCRIPTION
**Pull request by AndyTWF**

### CODE REVIEW

File: app/Services/NetworkDataService.php [hunk 1]
```diff
@@ -56,11 +56,62 @@ private function processClients(array $clients): void
...
+        try {
+            $aircraft = NetworkAircraft::firstOrCreate(
+                ['callsign' => $callsign],
+                array_merge(
+                    ['callsign' => $callsign],
+                    $details
+                )
+            );
+
+            if ($aircraft->wasRecentlyCreated) {
```
Comment (atoff) [inline app/Services/NetworkDataService.php:105]: Is this meant to be a negative instead? I'm thinking that if it was recently created then the timestamps should be fine?

File: app/Services/NetworkDataService.php [hunk 2]
```diff
@@ -56,11 +56,62 @@ private function processClients(array $clients): void
...
+                $aircraft->touch();
+            }
+        } catch (QueryException $queryException) {
+            if ($queryException->errorInfo[1] !== 1062) {
+                throw $queryException;
             }
-        });
+            $aircraft = NetworkAircraft::find($callsign);
```
Comment (atoff) [inline app/Services/NetworkDataService.php:112]: Do we want to perform a `touch` here too?

Comment (AndyTWF) [reply app/Services/NetworkDataService.php:112]: Happy to leave this as non-touched, if we ever hit this line, the other competing query should have sorted that for us :)

File: app/Services/NetworkDataService.php (hunk 1 above)
Comment (AndyTWF) [reply app/Services/NetworkDataService.php:105]: Actually this bit isn't required, as I set the `updated_at` column whenever the model is created :)

**Pull request merged**